Installation/CHANGES.md

```diff
@@ -2299,14 +2308,14 @@ Release date: October 2014
 -   Changes in the set of supported platforms:
     -   The Microsoft Windows Visual C++ compiler 2008 (VC9) is no
         longer supported since CGAL-4.5.
--   Since CGAL version 4.0, Eigen was the recommended third-party
+-   Since CGAL version 4.0, Eigen was the recommended third-party
```

Comment:
I had never seen an editor replacing non-breaking spaces into regular spaces! That is a bug.

Anyway, that cannot be merged. Those non-breaking spaces are here for a reason: it is bad to break a line between a word and a number following it.